### PR TITLE
Release 0.0.17.1 (bugfix backport)

### PR DESCRIPTION
**Pull request by bdice**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

xref: https://github.com/NVIDIA/numba-cuda/pull/86

### CODE REVIEW

File: recipe/recipe.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.9
```
Comment (jakirkham) [inline recipe/recipe.yaml]:
```suggestion
    - python {{ python_min }}
```

File: recipe/recipe.yaml [hunk 2]
```diff
@@ -18,11 +18,11 @@ build:
...
     - pip
     - setuptools
   run:
-    - python
+    - python >=3.9
```
Comment (jakirkham) [inline recipe/recipe.yaml]:
```suggestion
    - python >={{ python_min }}
```

File: recipe/recipe.yaml [hunk 3]
```diff
@@ -18,11 +18,11 @@ build:
...
-    - python
+    - python ${{ python_min }}
     - pip
     - setuptools
   run:
-    - python
+    - python >=${{ python_min }}
     - numba >=0.59.1
 
 tests:
```
Comment (jakirkham) [inline recipe/recipe.yaml:28]: Think we want something like this in the test section

```suggestion
tests:
  - requirements:
     run:
       - python ${{ python_min }}
       - pip
```

Basing this off [this example]( https://github.com/conda/ceps/blob/4f4ac7545d4ba42fb10f71e40ad66ec865daab2e/cep-0014.md?plain=1#L801-L803 )

Comment (bdice) [reply recipe/recipe.yaml:28]: Great. I wasn't sure how to handle tests in the new recipe format. Thanks for the suggestion.

File: recipe/recipe.yaml [hunk 4]
```diff
@@ -18,14 +18,18 @@ build:
...
+  - requirements:
+     run:
+       - python ${{ python_min }}
+       - pip
```
Comment (jakirkham) [inline recipe/recipe.yaml:32]: Sorry I goofed on the indenting. Think this should fix it

```suggestion
  - requirements:
      run:
        - python ${{ python_min }}
        - pip
```

File: recipe/recipe.yaml [hunk 5]
```diff
@@ -18,14 +18,18 @@ build:
...
   - python:
       imports:
       - numba_cuda
```
Comment (jakirkham) [inline recipe/recipe.yaml:35]: Looks like we would need to convert this to `script` as well

```suggestion
  - script:
      - python -m pip check
      - python -c "import numba_cuda"
```

Note: This includes the `pip_check` below, but was unable to grab that in the current diff

File: recipe/recipe.yaml [hunk 6]
```diff
@@ -18,17 +18,21 @@ build:
...
-      imports:
-      - numba_cuda
+  - requirements:
+      run:
+        - python ${{ python_min }}
+        - pip
+  - script:
+      - python -m pip check
+      - python -c "import numba_cuda"
       pip_check: true
```
Comment (bdice) [inline recipe/recipe.yaml:36]:
```suggestion
```

File: recipe/recipe.yaml [hunk 7]
```diff
@@ -1,15 +1,15 @@
 context:
   name: numba-cuda
   sdist_name: numba_cuda
-  version: "0.0.17"
+  version: "0.0.17.1"
```
Comment (jakirkham) [inline recipe/recipe.yaml:4]: Looks like we need this to workaround issue: https://github.com/prefix-dev/rattler-build/issues/1248

```suggestion
  version: "0.0.17.1"
  python_min: "3.9"
```

**Review by jakirkham — COMMENTED** [review]
Thanks Bradley! 🙏

Had a suggestion below

Then would try re-rendering again

**Comment by bdice** [comment]
@conda-forge-admin, please rerender

File: recipe/recipe.yaml [hunk 8]
```diff
@@ -18,18 +19,21 @@ build:
...
-      imports:
-      - numba_cuda
-      pip_check: true
+  - requirements:
+      run:
+        - python ${{ python_min }}
+        - pip
+  - script:
+      - python -m pip check
+      - python -c "import numba_cuda"
```
Comment (jakirkham) [inline recipe/recipe.yaml:36]: Missed this was [one list item]( https://github.com/conda/ceps/blob/4f4ac7545d4ba42fb10f71e40ad66ec865daab2e/cep-0014.md?plain=1#L801-L805 ) 🤦‍♂️

```suggestion
  - requirements:
      run:
        - python ${{ python_min }}
        - pip
    script:
      - python -m pip check
      - python -c "import numba_cuda"
```

**Comment by jakirkham** [comment]
@conda-forge-admin , please re-render

**Review by jakirkham — APPROVED** [review]
Thanks Bradley! 🙏

**Comment by jakirkham** [comment]
Looks like Azure didn't start when this was merged

Have [manually started Azure]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1104498&view=results )

**Pull request merged**